backend/vcs/syncing/syncing.go

```diff
@@ -382,13 +386,15 @@ func (s *Service) SyncWithPeer(ctx context.Context, device cid.Cid) error {
 		return err
 	}
 
-	resp, err := c.ListObjects(ctx, &p2p.ListObjectsRequest{})
+	remoteObjs, err := c.ListObjects(ctx, &p2p.ListObjectsRequest{})
 	if err != nil {
 		return err
 	}
-
+	if s.inboundDisable {
```

Comment:
I guess what you want here is to not syncing any object, right? Isn't just disabling syncing enough for this? There's already on option, I believe. If not, isn't it easier to return from this function early, instead of overwriting the ListObject response from a remote peer? If you don't want to sync with the peer, why even asking it for their objects? Or am I misunderstanding this?

Reply:
@burdiyan I wanted to sync outbound only. I.e I send documents but I don't receive documents. That is the reverse of what the gateway does. If I return early, then I will be disabling sync in both ways.

Reply:
@juligasa SyncWithPeer is only called to receive documents (inbound). So if you return early it won't attempt to sync with the remote peer, which is what you want, right? Why would it disable the outbound as well?